src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -78,16 +53,17 @@ const InfoBox: UISchemaElementComponent<{
         data-se={dataSe}
         className={`infobox-${messageClass.toLowerCase()}`}
       >
-        {message.title && (
-          <Typography
-            component="h2"
-            variant="h6"
-          >
-            {message.title}
-          </Typography>
-        )}
-        <WidgetMessageContainer message={message} />
-        {links.length > 0 && renderLinks()}
+        {
+          Array.isArray(message)
+            ? message.map((msg) => (
```

Comment:
`InfoBox` can now accept an array of messages to be able to display multiple messages inside one `InfoBox`